test/lib/pine-init.ts

```diff
@@ -0,0 +1,55 @@
+import * as express from 'express';
+import { exit } from 'process';
+import * as pine from '../../src/server-glue/module';
+
+export async function init(
+	initConfig: any,
```

Comment:
Can you type this better than any? I'm pretty sure there's a type for it (worst case scenario `Parameters<typeof pine.init>[1]` would match the type of where it gets passed into)

Reply:
Typed as ConfigLoader.Config